ores/lock_manager/poolcounter.py

```diff
@@ -49,5 +67,19 @@ def close(self):
             self.stream.close()
             self.stream = None
             return True
+
+        return False
+
+    @classmethod
+    def from_config(cls, config, name, section_key="lock_managers"):
+        primary_node_config = config[section_key][name]['primary_node']
+        secondary_node_config = config[section_key][name].get('secondary_node')
+        if secondary_node_config:
+            secondary_node = (secondary_node_config['server'],
+                              secondary_node_config['port'])
         else:
-            return False
+            secondary_node = None
+        return cls(
+            (primary_node_config['server'], primary_node_config['port']),
```

Comment:
It's anxiety-inducing that the primary and secondary clauses aren't parallel.

Reply:
Done now